src/pages/ParticipantsPage.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import OptionsList from '../components/OptionsList';
+import ROUTES from '../ROUTES';
+import CONST from '../CONST';
+
+const matchType = PropTypes.shape({
+    params: PropTypes.shape({
+        // reportID passed via route /participants/:reportID
+        reportID: PropTypes.string,
+    }),
+});
+
+const myPersonalDetailsType = PropTypes.shape({
+    // Display name of the current user from their personal details
+    displayName: PropTypes.string,
+
+    // Avatar URL of the current user from their personal details
+    avatar: PropTypes.string,
+
+    // login of the current user from their personal details
+    login: PropTypes.string,
+});
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    myPersonalDetails: myPersonalDetailsType.isRequired,
```

Comment:
For this one (and `matchType`) you don't need to make separate variables. Just put those inline here.